### DISCUSSION TITLE
AttributeError: module 'numpy.typing' has no attribute 'NDArray'

### DISCUSSION THREAD

**Discussion by mtopeW4EF** [scikit-rf/scikit-rf#1052] [Q&A] [answered]
I tried using the "autofit()" method for vector fitting and discovered it wasn't supported in version scikit-rf v0.20, so I upgraded to v0.32. Now I am getting a numpy error  (AttributeError: module 'numpy.typing' has no attribute 'NDArray') when I try importing scikit-rf into Python:

```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-51719e6feff2> in <module>
     28 from pylab import *
     29 #rf.stylely()
---> 30 import skrf as rf

~\Anaconda3\lib\site-packages\skrf\__init__.py in <module>
      9 
     10 
---> 11 from . import (
     12     calibration,
     13     circuit,

~\Anaconda3\lib\site-packages\skrf\calibration\__init__.py in <module>
     16 
     17 #from parametricStandard import *
---> 18 from . import calibration, calibrationSet, deembedding
     19 from .calibration import *
     20 from .calibrationSet import *

~\Anaconda3\lib\site-packages\skrf\calibration\calibration.py in <module>
    121 from ..networkSet import NetworkSet
    122 
--> 123 ComplexArray = npy.typing.NDArray[complex]
    124 
    125 global coefs_list_12term

AttributeError: module 'numpy.typing' has no attribute 'NDArray'
```

**Comment by Vinc0110** [comment]
Not sure, but maybe numpy needs an upgrade as well?

**Reply by mtopeW4EF** [reply, marked as answer]
It turned out that the fix was totally uninstalling and reinstalling Anaconda-Navigator. That fixed everything. 

By the way, the correct syntax for auto fitting is "vf.auto_fit()". In portions of the release notes for scikit-rf 0.32 it lists the syntax as "vf.autofit()": https://github.com/scikit-rf/scikit-rf/pull/1033


The syntax is correct in the scikit-rf documentation.

**Reply by jhillairet** [reply]
Thanks, the release note typo as been fixed.

**Reply by Henryplay** [reply]
This is because the version of numpy is not high enough, `pip uninstall numpy` and `pip install numpy` will solve it.
